test/acceptance/test/profiles_test.go

```diff
@@ -57,8 +58,12 @@ var _ = PDescribe("Weave GitOps Profiles API", func() {
 	})
 
 	AfterEach(func() {
-		deleteRepo(tip.appRepoName, gitproviders.GitProviderGitHub, githubOrg)
+		session := runCommandAndReturnSessionOutput(fmt.Sprintf("kubectl -n %s delete kustomizations --all", namespace))
+		Eventually(session, "20s", "1s").Should(gexec.Exit(0))
+		session = runCommandAndReturnSessionOutput(fmt.Sprintf("kubectl -n %s delete gitrepositories --all", namespace))
```

Comment:
Nice! 🙌  With this addition, we can unpend the Profiles API suite since the first `It` block should pass. We only need to pend the second `It` block (see https://github.com/weaveworks/weave-gitops/issues/1394). Could you try this locally please? :)

Reply:
giving it a go!

Reply:
green! for both tests :scream: